packages/app-headless-cms/src/admin/plugins/fieldRenderers/dateTime/Time.tsx

```diff
@@ -8,7 +8,7 @@ const Time = props => {
             {...props}
             field={{
                 ...props.field,
-                label: props.label
+                label: props.label || props.field.label
```

Comment:
Why do we need `props.label || props.field.label` ? 🤔

Reply:
`Time` component is used in two different places.

- As standalone `Time` field in case of `DateTime` format is set to `Time only`. The label is what we specify in `field settings`. 
- But, when `Time` is used to compose `DateTimeWithoutTimezone` and `DateTimeWithTimezone` it has label specified by that component. For example, if the field label is `sale ends` in the case of `DateTimeWithTimezone`, the label for `Time` field will be `sale ends time`.

Reply:
I was doing something wrong. It doesn't have to be this way.
Thanks for the feedback @doitadrian :)